src/instructlab/cli/model/train.py

```diff
@@ -384,6 +384,11 @@ def clickpath_setup(is_dir: bool) -> click.Path:
     is_flag=True,
     help="Optimize Memory Usage on CPU and MacOS. This uses the torch_dtype='auto' instead of float32",
 )
+@click.option(
+    "--disable-accelerate-full-state-at-epoch",
```

Comment:
If the purpose of this flag is to prevent full state at epoch (which assumes that full state at epoch is default, may not be the case in all scenarios I'm guessing), then would it make sense to simply expose the `--accelerate-full-state-at-epoch` setting as a flag, and having it be default to False? 

I see the utility in having this default to on in the profiles for large scenarios, however; I think it's currently being weighed down by the heavy requirement of storing all the checkpoints. If we have a policy to only keep the last full state checkpoint somehow then I think it would make sense to have `--full-state-at-epoch` be set to true.

Reply:
My first version of this did exactly as you say but I decided that the expected behavior of the flag was kinda confusing if it's 1:1. 

e.g. "User wants to _enable_ full state epochs--> _don't_ use `--full-state-at-epoch`" (defaults to True) vs.
"User wants to _disable_ full state epochs --> _use_ `--full-state-at-epoch`" (sets False)

It's kinda logically switched, so I changed it to the current version.

I do think that overwriting full state checkpoints is the correct default approach, though, so I'll open an issue for that as well. That behavior, comboed with this, seems reasonable to me.

Reply:
That makes sense. @cdoern Would you be okay with this? 

> I do think that overwriting full state checkpoints is the correct default approach, though, so I'll open an issue for that as well. That behavior, comboed with this, seems reasonable to me.

Yeah we can do this in a follow-up PR.

Reply:
^ created an issue for this

Reply:
I think this is fine because it's a flag not an option. if it was an option I'd be more particular but this is fine to me